docs/components/blockquote.md

```diff
@@ -9,10 +9,10 @@ The HTML blockquote tag implemented in ruby.
 This component can take 3 optional configuration params and either yield content or display what gets passed to the `text` configuration param.
 
 #### # id (optional)
-Expects a string with all ids the span should have.
+Expects a string with all ids the blockquote should have.
```

Comment:
Thank you a lot for noticing those and fixing them as well. For the future, may I ask for another PR on stuff like that so it gets properly separated from the `feature` that is the added `<q>` tag? :)

Reply:
This will not keep me from merging this time, but please take as advice for the future!

Reply:
Of course. I didn't know GitHub was going to add it to my first PR.